app/(playground)/p/[agentId]/beta-proto/giselle-node/components/panel/text-generator.tsx

```diff
@@ -140,6 +143,46 @@ export const TextGeneratorPropertyPanel: FC<
 									</SelectContent>
 								</Select>
 							</div>
+							<div className="border-t border-[hsla(222,21%,40%,1)]" />
+							<div className="grid gap-[16px]">
+								<div className="font-rosart text-[16px] text-black-30">
+									Parameters
+								</div>
+								<div className="grid gap-[16px]">
+									<TemperatureSlider
+										value={(node.properties.temperature as number) ?? 1.0}
+										onChange={(temperature) => {
+											dispatch(
+												updateNode({
+													input: {
+														nodeId: node.id,
+														properties: {
+															...node.properties,
+															temperature,
+														},
+													},
+												}),
+											);
+										}}
+									/>
+								</div>
+								<TopPSlider
+									value={(node.properties.topP as number) ?? 0.5}
```

Comment:
```suggestion
									value={(node.properties.topP as number) ?? 1.0}
```

Reply:
@toyamarinyon 

I think that for OpenAI API, `top_p` should be set to `1` since the default is `1`. 

https://platform.openai.com/docs/api-reference/chat/create#chat-create-top_p

<img width="632" alt="image" src="https://github.com/user-attachments/assets/38eaec3c-a4f0-4ed7-8e09-f3e673e8332b">

---

As for the Anthropic API, I read the documentation, but I couldn't understand it.

https://docs.anthropic.com/en/api/messages

Reply:
@shige Thanks! I've committed https://github.com/giselles-ai/giselle/pull/81/commits/1732ee087d3a4a8570798f56b1618a955890cc3e